AdAway/src/main/res/values-de/strings.xml

```diff
@@ -195,44 +193,44 @@
 
     <!-- Other -->
     <string name="app_subtitle">Werbeblocker</string>
-    <string name="no_text_editor_title">Keine Textbearbeitung installiert</string>
-    <string name="no_text_editor">Es konnte keine Textbearbeitung gefunden werden, um die Hosts-Datei zu öffnen. Sie können Jota oder eine andere Textbearbeitung installieren, um Textdateien zu öffnen.\n
+    <string name="no_text_editor_title">Kein Texteditor installiert</string>
+    <string name="no_text_editor">Es konnte kein Texteditor gefunden werden, um die Hostdatei zu öffnen. Sie können Jota oder andere Texteditoren installieren, um damit zu interagieren. \n
   \n
   Möchten Sie Jota installieren?</string>
-    <string name="no_file_manager_title">Keine Dateiverwaltung installiert</string>
-    <string name="no_file_manager">Es konnte keine Dateiverwaltung gefunden werden, um Dateien zu öffnen. Sie können »OI File Manager« oder  andere Dateimanager installieren, um das zu beheben.\n
-  \n
-   Möchten Sie »OI File Manager« installieren?</string>
+    <string name="no_file_manager_title">Kein Datei-Manager installiert</string>
+    <string name="no_file_manager">Es konnte kein Dateimanager zum Öffnen von Dateien gefunden werden. Sie können den OI-Dateimanager oder andere Dateimanager installieren, um damit zu agieren.\n
+    \n
+    Möchten Sie den OI-Dateimanager installieren?</string>
 
     <!-- Tcpdump -->
     <string name="toast_tcpdump_log_deleted">Protokolldatei wurde gelöscht!</string>
-    <string name="toast_tcpdump_added_to_blacklist">Der Hostname wurde zu Ihrer Negativliste hinzugefügt!</string>
-    <string name="toast_tcpdump_added_to_whitelist">Der Hostname wurde zu Ihrer Positivliste hinzugefügt!</string>
+    <string name="toast_tcpdump_added_to_blacklist">Hostname wurde zu Ihrer Negativliste hinzugefügt!</string>
+    <string name="toast_tcpdump_added_to_whitelist">Hostname wurde zu Ihrer Positivliste hinzugefügt!</string>
     <string name="tcpdump_log_context_blacklist">Zur Negativliste hinzufügen</string>
-    <string name="tcpdump_log_context_whitelist">Zur Positivliste hinzufügen</string>
+    <string name="tcpdump_log_context_whitelist">Zu Deiner Positivliste hinzufügen</string>
```

Comment:
Either revert or also change the string about `Negativliste` respectively. Should be unified.